### PR TITLE
Cherrypick: Fix vulnerabilities and CHANGELOG (#239)

### PR DESCRIPTION
**Pull request by pmarin-stratio**
###### Before submitting the PR, please make sure you do the following 
-  [ ] If there are changes to the backend codebase, run the backend tests with `lein test && lein eastwood && lein bikeshed && lein docstring-checker && ./bin/reflection-linter`
-  [ ] Run the frontend and integration tests with  `yarn lint && yarn flow && yarn test`)
-  [ ] Sign the [Contributor License Agreement](https://docs.google.com/a/metabase.com/forms/d/1oV38o7b9ONFSwuzwmERRMi9SYrhYeOrkbmNaq9pOJ_E/viewform)
(unless it's a tiny documentation change).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@
 * Restoring impersonation in queries related to Crossdata databases and tables
 * Update Crossdata JDBC driver to 2.19.0-809dcaf version
 
+## 0.39.1-68deaab (Built: November 04, 2020 | Released: November 04, 2020)
+
+* Bump import/export artifact version to 1.2.1-1570d16 (fix parentId query param)
+* Fix vulnerabilities
+
```
Comment (grios-stratio) [inline CHANGELOG.md:13]: Esto aquí no va, esto es solo de la rama 0.39

Comment (pmarin-stratio) [reply CHANGELOG.md:13]: te parece que los metamos en la 0.40 entonces??

**Review by Frannie-Ludmilla — APPROVED** [review]
LGTM :rocket:

**Review by grios-stratio — APPROVED** [review]

**Review by Frannie-Ludmilla — APPROVED** [review]
LGTM! :cherries:

**Pull request merged**